ir/expression.def

```diff
@@ -272,6 +272,7 @@ class Slice : Operation_Ternary {
 }
 
 class Member : Operation_Unary {
+    precedence = DBPrint::Prec_Postfix;
```

Comment:
Operation_Unary also includes Neg, Cmpl, LNot, which are prefixes, aren't they?

Reply:
Yes, but this field is just for Member.

Reply:
Oh for some reason I read this as the Operation Unary definition... sorry about that.